.github/workflows/release.yml

```
@@ -183,10 +183,20 @@ jobs:
         with:
           repository: beaver-lodge/kinda
           path: kinda
+      - name: Strip dev suffix
+        run: sed -i 's/-dev//g' mix.exs
       - name: Build ARM
-        continue-on-error: true
         if: ${{ matrix.image.platform == 'linux/arm64/v8' }}
         run: docker run --platform ${{ matrix.image.platform }} -v $PWD:/src -w /src/beaver ${{ vars.DOCKERHUB_USERNAME }}/beaver-livebook-${{ matrix.image.suffix }}:latest bash scripts/build-for-publish.sh
+      - name: Publish archives and packages
+        uses: softprops/action-gh-release@v1
```

actionlint (1.7.4)

the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)